### ISSUE TITLE
API message returned is error: error: unknown variable 'DATE'.

### ISSUE THREAD

**Issue by CVIOG-Demography** [walkerke/tidycensus#200]
The get_estimates function, when time_series is set to TRUE, still used "DATE" in the API call. The issue is that the API call has changed as of 2018 from DATE to DATE_CODE and PERIOD to PERIOD_CODE. Even when I specify a year prior to 2018, the same error is thrown. 

From the documentation:
The returned column is either "DATE", representing a particular estimate date, or "PERIOD",

From the CB:
https://www.census.gov/data/developers/data-sets/popest-popproj/popest/popest-vars/2018.html

pop18 = get_estimates(geography = "place",
                    product = "population",
                    county = "coweta",
                    state = "GA",
                    year = 2017,
                    time_series = TRUE,  
                    geometry = FALSE)

Thank you.

**Comment by mfherman** [comment]
Make sure you have the latest GitHub version of tidycensus installed. This API change was addressed in #170 

In addition, you can only access the time series data from the API by setting the most recent year available, 2018. @walkerke, perhaps the default year on `get_estimates()` should be bumped to 2018?

See below for examples with time series from 2018 and single year from 2017.

``` r
library(tidycensus)
get_estimates(geography = "place",
              product = "population",
              county = "coweta",
              state = "GA",
              year = 2018,
              time_series = TRUE,
              geometry = FALSE)
#> # A tibble: 220 x 5
#>    NAME                                        DATE GEOID    variable value
#>    <chr>                                      <dbl> <chr>    <chr>    <dbl>
#>  1 Chattahoochee Hills city (part), Coweta C…     1 1307715… POP         NA
#>  2 Chattahoochee Hills city (part), Coweta C…     2 1307715… POP          0
#>  3 Chattahoochee Hills city (part), Coweta C…     3 1307715… POP          0
#>  4 Chattahoochee Hills city (part), Coweta C…     4 1307715… POP          0
#>  5 Chattahoochee Hills city (part), Coweta C…     5 1307715… POP          0
#>  6 Chattahoochee Hills city (part), Coweta C…     6 1307715… POP          0
#>  7 Chattahoochee Hills city (part), Coweta C…     7 1307715… POP          0
#>  8 Chattahoochee Hills city (part), Coweta C…     8 1307715… POP          0
#>  9 Chattahoochee Hills city (part), Coweta C…     9 1307715… POP          0
#> 10 Chattahoochee Hills city (part), Coweta C…    10 1307715… POP          0
#> # … with 210 more rows


get_estimates(geography = "place",
              product = "population",
              county = "coweta",
              state = "GA",
              year = 2017,
              time_series = FALSE,
              geometry = FALSE)
#> # A tibble: 18 x 4
#>    NAME                                           GEOID     variable  value
#>    <chr>                                          <chr>     <chr>     <dbl>
#>  1 Grantville city, Coweta County, Georgia        13077344… POP       3226 
#>  2 Haralson town (pt.), Coweta County, Georgia    13077365… POP        181 
#>  3 Moreland town, Coweta County, Georgia          13077526… POP        434 
#>  4 Newnan city, Coweta County, Georgia            13077550… POP      38909 
#>  5 Palmetto city (pt.), Coweta County, Georgia    13077588… POP        311 
#>  6 Senoia city, Coweta County, Georgia            13077696… POP       4213 
#>  7 Sharpsburg town, Coweta County, Georgia        13077699… POP        361 
#>  8 Turin town, Coweta County, Georgia             13077777… POP        345 
#>  9 Balance of Coweta County, Coweta County, Geor… 13077999… POP      95134 
#> 10 Grantville city, Coweta County, Georgia        13077344… DENSITY    557.
#> 11 Haralson town (pt.), Coweta County, Georgia    13077365… DENSITY    250.
#> 12 Moreland town, Coweta County, Georgia          13077526… DENSITY    480.
#> 13 Newnan city, Coweta County, Georgia            13077550… DENSITY   2041.
#> 14 Palmetto city (pt.), Coweta County, Georgia    13077588… DENSITY    338.
#> 15 Senoia city, Coweta County, Georgia            13077696… DENSITY    732.
#> 16 Sharpsburg town, Coweta County, Georgia        13077699… DENSITY    360.
#> 17 Turin town, Coweta County, Georgia             13077777… DENSITY    272.
#> 18 Balance of Coweta County, Coweta County, Geor… 13077999… DENSITY    235.
```

<sup>Created on 2019-11-15 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

**Comment by walkerke** [comment]
This is now fixed; Census changed `DATE` to `DATE_` (trailing underscore) internally which was causing the error.
